action.yml

```
@@ -28,6 +35,6 @@ inputs:
   postCommands:
     description: "Commands to execute after publishing the Workers project"
     required: false
-  publish:
-    description: "Set to false to skip publishing your Worker project and secrets.  Defaults to true."
+  command:
+    description: "The Wrangler command you wish to run. For example: publish - this will publish your Worker"
```

Perhaps highlight that this will be a string?

```suggestion
    description: "The Wrangler command you wish to run. For example: `\"publish\"` - this will publish your Worker"
```

---

good call